docs/sdk/api/sdkMiddlewareAuth.md

```diff
@@ -60,12 +55,11 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle authentication for
 
 #### Named arguments (options)
 
-1. `host` *(String)*: the host of the OAuth API service
-2. `projectKey` *(String)*: the key of the project to assign the default scope to
-   - The user field is an object containining `username` and `password`. [Sample below](#usage-example)
-3. `credentials` *(Object)*: the client credentials for authentication (`clientId`, `clientSecret`)
-4. `scopes` *(Array)*: a list of [scopes](http://dev.commercetools.com/http-api-authorization.html#scopes) to assign to the OAuth token. _No default scope is sent_
-
+1. `host` _(String)_: the host of the OAuth API service
+2. `projectKey` _(String)_: the key of the project to assign the default scope to
+   * The user field is an object containining `username` and `password`. [Sample below](#usage-example)
```

Comment:
Thanks for this @adnasa 💯 🥇 
While we're at this, add the `user` on the next line, and move this line below it? Also, the `Sample below` link references the example from `clientCredentialsFlow` which you have refactored away. Could you fix this to link to the proper usage example? Thanks.

Reply:
> Also, the Sample below link references the example from clientCredentialsFlow which you have refactored away

I think you're looking at the wrong line. I refactored away users from `createAuthMiddlewareForClientCredentialsFlow` where as this link is referring to the example in `createAuthMiddlewareForPasswordFlow`

Reply:
The (#usage-example) should be (#usage-example-1) as they point to different usage examples